.devcontainer/scripts/notify-dev-entrypoint.sh

```diff
@@ -16,6 +16,12 @@ echo -e "alias l='exa -alh'" >> ~/.bashrc
 echo -e "alias ll='exa -alh@ --git'" >> ~/.bashrc
 echo -e "alias lt='exa -al -T -L 2'" >> ~/.bashrc
 
+# Kubectl aliases and command autocomplete
+echo -e "alias k='kubectl'" >> ~/.bashrc
+echo -e "source <(kubectl completion bash)" >> ~/.bashrc
+echo -e "complete -F __start_kubectl k" >> ~/.bashrc
+echo -e "source /usr/share/bash-completion/bash_completion" >> ~/.bashrc
```

Comment:
Could be nice to use oh my zsh instead, dump starship. The former offers so much for auto completion and plugins.

Reply:
I'm definitely game for switching to `zsh`, I had just thought there was a `starship` preference.  I'll get that added!

Reply:
I like starship as it's cross platform and written in Rust (I contributed a bit to it) -- but oh-my-zsh might play much better for devcontainers and all of its plugins. 🙃